test/fiddle/test_closure.rb

```diff
@@ -6,6 +6,12 @@
 
 module Fiddle
   class TestClosure < Fiddle::TestCase
+    def setup
+      if RUBY_ENGINE == "truffleruby"
+        omit("FFI::Function don't accept #call-able object with TruffleRuby")
```

Comment:
I'll look into this, TruffleRuby does pass most FFI gem tests so it's surprising, maybe this is not well tested or is not an officially supported API.

Reply:
Fixed in https://github.com/ruby/fiddle/pull/149/commits/41c21f5887af6133ed5ee347ac6143cbc396c382 so all such tests pass now